Add support for component tests

I suppose the plugin is not expected to work with component tests. I assume that because the README doesn't explicitly say that and moreover the plugin doesn't work for component tests. It throws an error:
```
Cannot find module 'path'
```

Would it be difficult to make the plugin work for component tests?

Cypress version: 13.11
Node: 18

--- a/package.json
+++ b/package.json
@@ -31,6 +31,7 @@
     "image-size": "^1.1.1",
     "lodash": "^4.17.21",
     "odiff-bin": "^3.0.0",
+    "path-browserify": "^1.0.1",
     "pluralize": "^8.0.0"
   },
   "devDependencies": {

--- a/src/support/commands.ts
+++ b/src/support/commands.ts
@@ -1,4 +1,4 @@
-import * as path from 'path'
+import * as path from 'path-browserify'
 
 const goldImages = 'cypress/gold'
 

